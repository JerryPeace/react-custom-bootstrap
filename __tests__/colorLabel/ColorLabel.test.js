import React from 'react';
import { mount } from 'enzyme';
import toJSON from'enzyme-to-json';
import ColorLabel from 'colorLabel/index';

describe('ColorLabel', () => {
    test('render productLogo correctly', () => {
        const colorLabel = mount(
            <ColorLabel>
                <div>test</div>
            </ColorLabel>
        );
        expect(toJSON(colorLabel)).toMatchSnapshot();
    });
});
