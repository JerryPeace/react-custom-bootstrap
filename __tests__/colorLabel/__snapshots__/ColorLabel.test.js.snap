// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorLabel render productLogo correctly 1`] = `
<ColorLabel
  bsClass="label"
  bsStyle="light-gray"
>
  <span
    className="label label-light-gray"
  >
    <div>
      test
    </div>
  </span>
</ColorLabel>
`;
